test/fixtures/config/.soycriticrc

```diff
@@ -1,4 +1,8 @@
 {
   "callToImportRegex": "(\\S+)",
-  "callToImportReplace": "{$1|param}"
+  "callToImportReplace": "{$1|param}",
+  "implicitParams": {
+    "*Component": ["additionalParam"],
+    "Foo": ["bar"]
```

Comment:
I think this is leftover from some tests I was conducting... we can delete it, or make it useful in conjunction with the rest of the test configuration.